public/website.php

```diff
@@ -59,11 +59,7 @@
 // Comment this line if you want to use the "varnish" http
 // caching strategy. See http://sulu.readthedocs.org/en/latest/cookbook/caching-with-varnish.html
 if ($env !== 'dev') {
-    $kernel = new SuluHttpCache($kernel);
-
-    // When using the HttpCache, you need to call the method in your front controller
-    // instead of relying on the configuration parameter
-    Request::enableHttpMethodParameterOverride();
```

Comment:
could not find any reference to this in the foshttpcache docs so I removed  /cc @trickreich

Reply:
found the reference in the symfony cache kernel docs so I readded it. I would always activate it to have no difference between request on dev or prod